spec/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
- [ ] Kindly remove this pending test.

- [ ] **[OPTIONAL]** I think it would be a better idea to move this test into a `spec/models` folder, instead of just leaving it as `spec` as it makes for a better organization.